html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <label for="Email">Email:</label>
+      <input id="Email" name="Email" type="text">
+      <div>
+        <label for="Why is coding cool?">Why is coding cool?</label>
+      </div>
+      <select name="Why is coding cool?">
+        <option value="select">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+      <div>
+        <p>Favorite Language</p>
+        <input id="HTML" type="radio" name="Favorite Language" value="HTML">
+        <label for="HTML">HTML</label>
+        <input id="CSS" type="radio" name="Favorite Language" value="CSS">
+        <label for="CSS">CSS </label>
+        <input id="JavaScript" type="radio" name="Favorite Language" value="Javascript">
+        <label for="JavaScript">Javascript</label>
+      </div>
+      <div>
+        <label for="checkbox"></label>
+        <p>Would you like to sign up for more information?</p>
+        <div>
+          <input id="checkbox" type="checkbox" name="more information" value="yes">
+          <label for="checkbox">Sign me up!</label>
+        </div>
+      </div>
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <textarea name="comments" cols="25" rows="10"></textarea>
```

Comment:
The value of the `id` attribute on the `input` needs to match the value of the `for` attribute on the `label` it corresponds with.